luigi/task.py

```diff
@@ -218,12 +218,12 @@ def disable_window(self):
         Override this positive integer to have different ``disable_window`` at task level.
         Check :ref:`scheduler-config`
         """
-        return self.disable_window_seconds
+        return None
 
     @property
     def disable_window_seconds(self):
         warnings.warn("Use of `disable_window_seconds` has been deprecated, use `disable_window` instead", DeprecationWarning)
-        return None
+        return self.disable_window
```

Comment:
Does this line need a test added to get Codecov to go green? Happy to add it if required.

Reply:
I think this is what `Codecov` is complains about. If you can fix it, it would awesome.

Reply:
which skim looks like only the scheduler's `disable_window` param is unittested. I don't immediately see any of the Task's. Having test coverage would be great if there's valuable logic that be added to verify. I won't block approval and merge without it. Let me know what you want to do.

Reply:
Have added a very basic test on `disable_window_seconds` which (hopefully) will give the required coverage 🤞🏻